source/docs/v3/tenants.blade.md

```diff
@@ -130,6 +130,23 @@ You may access the current tenant using the `tenant()` helper. You can also pass
 
 Alternatively, you may typehint the `Stancl\Tenancy\Contracts\Tenant` interface to inject the model using the service container.
 
+## Accessing the central application {#accessing-the-central-application}
+
+When your code executes in the context of a tenant (for instance, when the `DatabaseMigrated` event is dispatched), you may access the central application context by using the `tenancy()->central($callback)` function.
```

Comment:
I don't think the point about `DatabaseMigrated` is needed here, it probably adds confusion.